docs/UserGuide/Cluster/Cluster-Maintenance.md

```diff
@@ -82,6 +82,8 @@ The ConfigNode statuses are defined as follows:
 
 - **Running**: The ConfigNode is running properly.
 - **Unknown**: The ConfigNode doesn't report heartbeat properly.
+  - Can't receive data synchronized from other ConfigNodes
```

Comment:
synchronized is adj. or v.

Reply:
This is an attributive clause that leaves out that~